app/src/main/java/com/codesoom/assignment/TaskHttpHandler.java

```diff
@@ -0,0 +1,180 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.models.Task;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class TaskHttpHandler implements HttpHandler {
+    private final List<Task> tasks = new ArrayList<>();
+    int statusCode;
+
+    @Override
+    public void handle(HttpExchange httpExchange) throws IOException {
+        String method = httpExchange.getRequestMethod();
+        URI uri = httpExchange.getRequestURI();
+        String path = uri.getPath();
+
+
+        InputStream inputStream = httpExchange.getRequestBody();
+
+
+        System.out.println(method + " " + path);
+
+        String content = "Hello, world!";
+
+        Long pathId = Long.parseLong(getPathId(path));
+        try {
+            statusCode = 200;
+            if (method.equals("GET") && path.equals("/tasks/" + pathId)) {
+                content = taskToJson(pathId);
+            }
+
+            if (method.equals("GET") && path.equals("/tasks")) {
+                content = tasksToJson();
+            }
+
+            if (method.equals("POST") && path.equals("/tasks")) {
+                Task task = toTask(getBody(inputStream));
+                tasks.add(task);
+                content = task.toString();
+                statusCode = 201;
+            }
+
+            if (method.equals("PUT") && path.equals("/tasks/" + pathId)) {
+                content = modifyTaskById(getBody(inputStream), pathId);
+            }
+
+            if (method.equals("DELETE") && path.equals("/tasks/" + pathId)) {
+                content = deleteTaskById(pathId);
+            }
+
+        } catch (IllegalArgumentException e) {
+            e.printStackTrace();
+            statusCode = 400;
+            content = "옳바른 요청이 아닙니다.";
+        } catch (NoSuchElementException e) {
+            e.printStackTrace();
+            statusCode = 404;
+            content = "ID에 해당하는 Task가 존재하지 않습니다.";
```

Comment:
보안 문제가 없고, `id가 `null`이 아니라면 이렇게 알려줄 수도 있을 겁니다. 물론 지금 코드에서는 어려울 수 있겠죠. 하지만 나중에는 이렇게도 할 수 있어야 합니다.

```suggestion
            content = "ID [" + id + "]에 해당하는 Task가 존재하지 않습니다.";
```

Reply:
id 라는 정보를 줌으로써 해당 메세지만 봐도 한번에 알수있을꺼같습니다.